src/Exception/RefreshTokenException.php

```diff
@@ -20,6 +20,9 @@
 
 namespace PrestaShop\Module\PsAccounts\Exception;
 
+/**
+ * @deprecated Replaced by \PrestaShop\Module\PsAccounts\Account\Exception\RefreshTokenException
+ */
 class RefreshTokenException extends \Exception
```

Comment:
We cannot remove it ?

Reply:
getShopToken is officially available since 7.1.1 and it's already in the docs... but probably not used